spec/controllers/questions/start_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Questions::StartController do
+  describe '#submit' do
+    it 'stores valid responses to the session' do
+      post :submit, params: { start_form: { priority_repair: 'Yes' } }
```

Comment:
Is this the value it actually gets from the view or will it be lowercase?

Reply:
Good spot :+1: